server/sched.go

```diff
@@ -133,10 +133,6 @@ func (s *Scheduler) processPending(ctx context.Context) {
 				numParallel = 1
 				slog.Warn("multimodal models don't support parallel requests yet")
 			}
-			// Keep NumCtx and numParallel in sync
-			if numParallel > 1 {
```

Comment:
This can be removed because:

1. We always set pending.opts.NumCtx based on `numParallel` below
2. We divide by numParallel in `needsReload` for a fair comparison